src/Features/CSharp/Portable/Completion/CompletionProviders/DeclarationNameCompletionProvider.cs

```diff
@@ -178,24 +179,30 @@ private static Glyph GetGlyph(SymbolKind kind, Accessibility? declaredAccessibil
 
             if (type is INamedTypeSymbol namedType && namedType.OriginalDefinition != null)
             {
-                var originalDefinition = namedType.OriginalDefinition;
-
-                var ienumerableOfT = namedType.GetAllInterfacesIncludingThis().FirstOrDefault(
-                    t => t.OriginalDefinition.SpecialType == SpecialType.System_Collections_Generic_IEnumerable_T);
+                var getEnumeratorMethod = namedType.GetMembers()
```

Comment:
When `namedType` is a `Span<T>`, `GetMembers` returns an empty immutable array. Why it doesn't return the existing GetEnumerator and all other methods?

Could someone assist here?